examples/main.tf

```diff
@@ -1,6 +1,9 @@
-module alert {
-  source                = "../"
-  monitoring_project_id = "hiiretail-monitoring-prod-6500"
-  notification_channels = ["projects/testproject/notificationChannels/channel_id"]
-  policies              = yamldecode(file("alerts.yaml"))
+module "alert" {
+  source                        = "../"
+  project                       = "<my-project-id>"
+  policies                      = yamldecode(file("alerts.yaml"))
+  default_notification_channels = ["#all-alerts-will-alert-to-this-slack-channel"]
```

Comment:
Is the plan to hardcode these in `terragrunt.hcl` file? or do you want to use dependencies to get them automatically?